### PR TITLE
Announce pulumi-gcp v8 release

### PR DESCRIPTION
**Pull request by guineveresaenger**
Draft for the GCP v8 release announcement. 

I'm not sure how to handle the frontmatter date, and would appreciate any help finding some fun images or graphics to make this look a little nicer.

### CODE REVIEW

File: content/blog/gcp-v8-release/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## What’s New in 8.0
+
+### Looking Back
+
+Since the last major release of this provider, we have continuously shipped improvements to our ecosystem, bringing the latest Pulumi features to your production stack.
+We have rolled out an improved diffing strategy in the Pulumi Terraform Bridge, removing spurious diffs on preview and increasing confidence when deploying.
+Additionally we have worked hard to improve accuracy and coverage for registry documentation.
+
+### Notable upstream features
```
Comment (mjeffryes) [inline content/blog/gcp-v8-release/index.md:38]: I'd suggest that this header should be the "What’s New in 8.0"

File: content/blog/gcp-v8-release/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- [Setup & Install](https://www.pulumi.com/registry/packages/gcp/installation-configuration/) - Instructions on installing the Google Cloud provider
+- [How-to guides](https://www.pulumi.com/registry/packages/gcp/how-to-guides/) - Learn how to use the Google Cloud provider to provision specific resources
+- [Templates](https://www.pulumi.com/templates/serverless-application/gcp/) - Use a quickstart template to create a new project
+- [Pulumi AI](https://www.pulumi.com/ai) - Ask Pulumi AI to create a new project
+
+## What’s New in 8.0
+
+### Looking Back
+
+Since the last major release of this provider, we have continuously shipped improvements to our ecosystem, bringing the latest Pulumi features to your production stack.
```
Comment (mjeffryes) [inline content/blog/gcp-v8-release/index.md:34]: Can we pull any stats on # or resources/functions/inputs like we did for last year?

File: content/blog/gcp-v8-release/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+We have rolled out an improved diffing strategy in the Pulumi Terraform Bridge, removing spurious diffs on preview and increasing confidence when deploying.
+Additionally we have worked hard to improve accuracy and coverage for registry documentation.
```
Comment (mjeffryes) [inline content/blog/gcp-v8-release/index.md:36]: Suggest we try to back these claims up with some links to notable improvements like 
[Improved example converter · pulumi-gcp/1266](https://github.com/pulumi/pulumi-gcp/pull/1266)
[[EPIC] Rollout PlanResourceChange· pulumi-terraform-bridge/1785](https://github.com/pulumi/pulumi-terraform-bridge/issues/1785)
[Fix bridge not running state upgrades from 0 -> 1 · pulumi-terraform-bridge/2081](https://github.com/pulumi/pulumi-terraform-bridge/pull/2081)

Comment (mjeffryes) [inline content/blog/gcp-v8-release/index.md:36]: Sounds more professional to drop the "worked hard to" IMO

```suggestion
We have rolled out an improved diffing strategy in the Pulumi Terraform Bridge, removing spurious diffs on preview and increasing confidence when deploying.
Additionally, we have improved accuracy and coverage for registry documentation.
```

**Review by mjeffryes — COMMENTED** [review]
Thanks Guin! Left a few suggestions; maybe @mnlumi still has some assets we could crib from to build a new version of the resource count image?

File: content/blog/gcp-v8-release/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Pulumi Google Cloud Provider Version 8.0.0"
+date: 2024-09-16T10:08:42-07:00
+draft: false
+meta_desc: "Release of the next version of the Pulumi Provider for Google Cloud"
+meta_image: meta.png
+authors:
+    - guineveresaenger
```
Comment (interurban) [inline content/blog/gcp-v8-release/index.md:8]:
```suggestion
    - guinevere-saenger
```

File: content/blog/gcp-v8-release/index.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    - release
+---
+
+The latest major release of the Pulumi Google Cloud Provider is available now!
+Our 8.0 release contains the latest upstream changes to keep you up-to-date along with the latest features and improvements from Pulumi.
+
+The Pulumi Google Cloud provider can be used to provision any of the Google Cloud resources available in the upstream provider.
+The provider is open source and available on GitHub so you can always follow along with current issues and developments, or even open your first pull request.
+
+<!--more-->
```
Comment (interurban) [inline content/blog/gcp-v8-release/index.md:20]: Are you adding more?

File: content/blog/gcp-v8-release/index.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+### Looking Back
+
+Since the last major release of this provider, we have continuously shipped improvements to our ecosystem, bringing the latest Pulumi features to your production stack.
+We have rolled out an improved diffing strategy in the Pulumi Terraform Bridge, removing spurious diffs on preview and increasing confidence when deploying.
+Additionally we have worked hard to improve accuracy and coverage for registry documentation.
+
+### Notable upstream features
+
+Several resources have new `deletionProtection` fields:
```
[the remark this answers is not in the document]
Comment (interurban) [reply content/blog/gcp-v8-release/index.md:40]: should we link to the registry docs here?

File: content/blog/gcp-v8-release/index.md (hunk 5 above)
Comment (guineveresaenger) [reply content/blog/gcp-v8-release/index.md:20]: I thought this was the fold for the blog post format. Isn't the "more" anything below this intro?

File: content/blog/gcp-v8-release/index.md (hunk 2 above)
Comment (guineveresaenger) [reply content/blog/gcp-v8-release/index.md:32]: this is the type of thing I'd need help with, I don't entirely know where to get started.

Comment (mjeffryes) [reply content/blog/gcp-v8-release/index.md:32]: I used some jq to pull these from the schema:

```
❯ cat provider/cmd/pulumi-resource-gcp/schema.json| jq '.functions | length '
319
❯ cat provider/cmd/pulumi-resource-gcp/schema.json| jq '.resources | length' 
981
❯ cat provider/cmd/pulumi-resource-gcp/schema.json| jq '.resources | map(.properties | length) | add'
9302
```

Can use the graphic from the last release for the stats on the prior version. I'm not sure how to make the pretty pictures though.

File: content/blog/gcp-v8-release/index.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Pulumi Google Cloud Provider Version 8.0.0"
+date: 2024-09-18T10:08:42-07:00
```
Comment (interurban) [inline content/blog/gcp-v8-release/index.md:3]:
```suggestion
date: 2024-09-19T10:08:42-07:00
```

**Review by interurban — APPROVED** [review]
Looks good, just made 1 suggestion to publish date (assuming its going today)

**Pull request merged**